src/pages/iou/IOUTransactions.js

```diff
@@ -88,6 +128,9 @@ class IOUTransactions extends Component {
                                 action={reportAction}
                                 key={reportAction.sequenceNumber}
                                 canBeRejected={canBeRejected}
+                                currentUserName={currentUserName}
+                                participantName={participantName}
+                                sessionEmail={sessionEmail}
```

Comment:
Same issue as before. We are building these props in this component only to pass them to the child instead of doing the logic in the component that needs this stuff. Maybe there is some reason why?

Reply:
`IOUTransactions` is a list component with `ReportTransactions` called inside a loop. I thought it'll be better to fetch these values once than to use the process multiple times in the child component.

Reply:
Got it. That makes sense to me. I missed that it was being called in a loop.